src/batched/KokkosBatched_Axpy_Decl.hpp

```diff
@@ -0,0 +1,66 @@
+#ifndef __KOKKOSBATCHED_AXPY_DECL_HPP__
+#define __KOKKOSBATCHED_AXPY_DECL_HPP__
+
+
+/// \author Kim Liegeois (knliege@sandia.gov)
+
+#include "KokkosBatched_Util.hpp"
+#include "KokkosBatched_Vector.hpp"
+
+namespace KokkosBatched {
+
+  ///
+  /// Serial AXPY
+  ///
+  ///
+  /// y <- alpha * x + y
+  ///
+  ///
+
+  struct SerialAxpy {
+    template<typename ViewType,
```

Comment:
Can the View and AlphaView different memory space? I thought X and Y will also be in scratchpad for a batched CG. Is there an use case for X and Y to be on global memory?

Reply:
Do you ask why `ViewType` and `alphaViewType` can be different?
If yes, it is not motivated by the memory space but by the used type.
In the case of the CG for instance, the scalar type of `alpha` will typically be a magnitude type and `X` and `Y` can store values with a different types.

Reply:
Good point. Now I have to check if that is what standard blas does :)